### PR TITLE
Set up Issue template

### LINKED ISSUE #694: Set up issue template
I'm tempted to convert all issues to discussions to remove the confusion between Issue, PR, and Discussion. However, I'm not sure that is a good idea.

Another approach is to create an issue template which makes it more clear what is required before creating an issue:
- PR that reproduces the issue using this repository's example site
- a failing test case
- a sample repository and steps to reproduce the issue

I don't want to prevent people from raising up issues they find, but it can be hard to replicate bugs sometimes because there are so many variables at play, i.e. Django settings, different browsers, different versions of Python, etc.

### PR DESCRIPTION
**Pull request by mihrab34**
to close #694

### CODE REVIEW

**Comment by mihrab34** [comment]
@adamghill I have some commits from my `origin main branch`. My local branch is up to date with the upstream and origin. Is there a way I could clean the commit history up? I'm thinking `git rebase`

**Comment by adamghill** [comment]
Yes! I think a rebase and force push should do it. If it doesn't work we can take a look at it tomorrow. 👍

**Comment by mihrab34** [comment]
@adamghill

File: .github/workflows/coverage.yml
Comment (adamghill) [inline .github/workflows/coverage.yml:1]: Is this from the other PR? Maybe a rebase will clear this out?

**Review by adamghill — CHANGES_REQUESTED** [review]
The issue templates look great! If you can clear out the extraneous commits that would be awesome, otherwise I might be able to clean them up. Let me know!

File: .github/workflows/coverage.yml
Comment (mihrab34) [reply .github/workflows/coverage.yml:1]: yes, got that after merging my `origin main` with my current feature branch. Had to clean it up with rebase

**Comment by mihrab34** [comment]
> The issue templates look great! If you can clear out the extraneous commits that would be awesome, otherwise I might be able to clean them up. Let me know!

@adamghill  I squashed all of them into 1 and fixed the part with the merge conflict.

**Review by adamghill — APPROVED** [review]
🚀

**Pull request merged**